test/unit/parser_test.rb

```diff
@@ -43,7 +43,7 @@ def test_inline_comments_in_input_file
     parser.advance
 
     # assert_equal("1234", parser.symbol)
-    # assert_equal(:A_COMMAND, parser.command_type)
+    assert_equal(:C_PUSH, parser.command_type)
```

Comment:
There’s another assertion in an earlier test (`# assert_equal(:C_COMMAND, parser.command_type)`) that should also be uncommented as part of this commit.